v2/internal/controllers/kubernetesconfiguration_fluxconfiguration_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package controllers_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	kubernetesconfiguration "github.com/Azure/azure-service-operator/v2/api/kubernetesconfiguration/v1api20230501"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+)
+
+func Test_KubernetesConfiguration_FluxConfiguration_CRUD(t *testing.T) {
+	t.Parallel()
+
+	tc := globalTestContext.ForTest(t)
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	adminUsername := "adminUser"
```

Comment:
Maybe not the most important topic in this PR, so feel free to ignore it or maybe we should file a separate test improvement issue, but I _think_ you can create AKS clusters without SSH access, and just specify managedIdentity intead.

Feels cleaner for these cases where don't actually care about SSH access?

Reply:
Sounds like a good idea, although bit torn on if it's worth spending time to get it replaced with MI and re-record the tests? 
Since it's just a CRUD test for a resource and we have random generated passwords safe.

Reply:
Maybe just file a low-priority issue for it?